### ISSUE TITLE
fnv1a_ch graphite-web compatibility 

### ISSUE THREAD

**Issue by druchoo** [grobian/carbon-c-relay#183]
In #44 @deniszh states that fnv1a_ch is not compatible with graphite-web. Is this still the case?

**Comment by Civil** [comment]
If you want to use graphite-web's clustering - yes, it's still the case because nobody implemented fnv1a or jump_fnv1a hashes in graphite-web's.

**Comment by druchoo** [comment]
Is it a factor in configuration where 1 graphite host runs multiple carbon-cache's?

**Comment by Civil** [comment]
Doesn't matter. If I'm correct, graphite-web uses it's implementation of consistent hashing to access cache's of carbon-cache (CARBON_LINK). As distribution is different for other types of hashs, it won't work correctly. But it doesn't mean that you can't use graphite-web with carbon-c-relay - you'll just loose ability to use CARBON_LINK.

**Comment by druchoo** [comment]
TL;DR: If using carbon's CARBON_LINK setting, use carbon_ch...
